bokehjs/src/lib/models/tools/index.ts

```diff
@@ -1,38 +1,40 @@
-export {CustomJSHover}     from "./inspectors/customjs_hover"
-export {HoverTool}         from "./inspectors/hover_tool"
-export {InspectTool}       from "./inspectors/inspect_tool"
-export {Tool}              from "./tool"
-export {ToolProxy}         from "./tool_proxy"
-export {Toolbar}           from "./toolbar"
-export {ToolbarBase}       from "./toolbar_base"
-export {ProxyToolbar}      from "./toolbar_box"
-export {ToolbarBox}        from "./toolbar_box"
+export { ActionTool }            from "./actions/action_tool"
+export { CustomAction }          from "./actions/custom_action"
+export { HelpTool }              from "./actions/help_tool"
+export { RedoTool }              from "./actions/redo_tool"
+export { ResetTool }             from "./actions/reset_tool"
+export { SaveTool }              from "./actions/save_tool"
+export { UndoTool }              from "./actions/undo_tool"
+export { ZoomInTool }            from "./actions/zoom_in_tool"
+export { ZoomOutTool }           from "./actions/zoom_out_tool"
+export { ButtonTool }            from "./button_tool"
+export { EditTool }              from "./edit/edit_tool"
+export { BoxEditTool }           from "./edit/box_edit_tool"
+export { FreehandDrawTool }      from "./edit/freehand_draw_tool"
+export { PointDrawTool }         from "./edit/point_draw_tool"
+export { PolyDrawTool }          from "./edit/poly_draw_tool"
+export { PolyTool }              from "./edit/poly_tool"
+export { PolyEditTool }          from "./edit/poly_edit_tool"
+export { BoxSelectTool }         from "./gestures/box_select_tool"
+export { BoxZoomTool }           from "./gestures/box_zoom_tool"
+export { GestureTool }           from "./gestures/gesture_tool"
+export { LassoSelectTool }       from "./gestures/lasso_select_tool"
+export { PanTool }               from "./gestures/pan_tool"
+export { PolySelectTool }        from "./gestures/poly_select_tool"
+export { RangeTool }             from "./gestures/range_tool"
+export { SelectTool }            from "./gestures/select_tool"
+export { TapTool }               from "./gestures/tap_tool"
+export { WheelPanTool }          from "./gestures/wheel_pan_tool"
+export { WheelZoomTool }         from "./gestures/wheel_zoom_tool"
+export { CrosshairTool }         from "./inspectors/crosshair_tool"
+export { CustomJSHover }         from "./inspectors/customjs_hover"
+export { HoverTool }             from "./inspectors/hover_tool"
+export { InspectTool }           from "./inspectors/inspect_tool"
+export { Tool }                  from "./tool"
+export { ToolProxy }             from "./tool_proxy"
+export { Toolbar }               from "./toolbar"
+export { ToolbarBase }           from "./toolbar_base"
+export { ProxyToolbar }          from "./toolbar_box"
+export { ToolbarBox }            from "./toolbar_box"
+export { Indicator }             from "./indicators/indicator"
+export { ServerStatusIndicator } from "./indicators/server_status_indicator"
```

Comment:
Our codebase standard is not to have leading/trailing space between `{}`. This will result in diff noise that is otherwise unnecessary. Please adjust here and in other places.

Reply:
Changed it, but there's still some noise here regardless. The `from` keywords were aligned previously, and the line for the new export `ServerStatusIdicator` uses more columns than where the `from` columns were before.

Three options:
1. All lines are affected by realigning the `from` column
2. All lines are affected by removing any alignment
3. Only the additions are modified, but the file has some lines aligned on the `from` keyword, and some others not aligned